### DISCUSSION TITLE
no constants exports in [@types/node]

### DISCUSSION THREAD

**Discussion by forest1102** [DefinitelyTyped/DefinitelyTyped#69244] [Issues with a @types package] [answered]
In this [release](https://nodejs.org/en/blog/release/v20.12.0#vm-support-using-the-default-loader-to-handle-dynamic-import) page, 
vm now has `constants.https://nodejs.org/en/blog/release/v20.12.0#vm-support-using-the-default-loader-to-handle-dynamic-importUSE_MAIN_CONTEXT_DEFAULT_LOADER` from v20.12.0. 
However, this does not apply to @types/node.

[a comment by an automated account is not shown]

**Comment by ghost** [comment, marked as answer]
Added in #69212
